tgui/packages/tgui/interfaces/FaxMachine.js

```diff
@@ -0,0 +1,68 @@
+import { useBackend } from "../backend";
+import { Button, LabeledList, Section } from "../components";
+import { Window } from "../layouts";
+
+export const FaxMachine = (props, context) => {
+  const { act, data } = useBackend(context);
+  return (
+    <Window>
+      <Window.Content>
+        <Section title="Authorization">
+          <LabeledList>
+            <LabeledList.Item label="ID Card">
+              <Button
+                icon={data.scan_name ? 'eject' : 'id-card'}
+                selected={data.scan_name}
+                content={data.scan_name ? data.scan_name : '-----'}
+                tooltip={data.scan_name ? "Eject ID" : "Insert ID"}
+                onClick={() => act("scan")} />
+            </LabeledList.Item>
+            <LabeledList.Item label="Authorize">
+              <Button
+                icon={data.authenticated ? 'sign-out-alt' : 'id-card'}
+                selected={data.authenticated}
+                disabled={!data.scan_name && !data.authenticated}
+                content={data.authenticated ? 'Log Out' : 'Log In'}
+                onClick={() => act("auth")} />
+            </LabeledList.Item>
+          </LabeledList>
+        </Section>
+        <Section title="Fax Menu">
+          <LabeledList>
+            <LabeledList.Item label="Network">
+              {data.network}
+            </LabeledList.Item>
+            <LabeledList.Item label="Currently Sending">
```

Comment:
Rename this to something like `Document` or `Item`. `Currently Sending` looks like the header for what the fax machine is literally already sending at that moment in time (IE: Its in the process of sending)